scripts/sign.py

```diff
@@ -4,55 +4,99 @@
 #
 # SPDX-License-Identifier: BSD-2-Clause
 
-
 def uuid_parse(s):
     from uuid import UUID
     return UUID(s)
 
-
 def int_parse(str):
     return int(str, 0)
 
 
 def get_args():
+    import sys
+    print(sys.argv)
     from argparse import ArgumentParser
-
-    parser = ArgumentParser()
+    command_choices = ['sign', 'generate-digest', 'stitch-ta']
+
+    parser = ArgumentParser(description='Sign a Tusted Application for OP-TEE.',usage='\n   ' +
+        '%(prog)s command [ command_opts ] command_args\n\n'
+
+    '   command:\n' +
+    '     sign             Generate signed loadable TA image file.\n' +
+    '                      Takes arguments --uuid, --in, --out and --key.\n' +
+    '     generate-digest  Generate loadable TA binary image digest for offline\n' +
```

Comment:
minor: maybe shorten `generate-digest` into `digest`.

maybe also `stitch-ta` into `stitch`, no?

Reply:
I think these are more clear in this verbose form, also aligns better with https://github.com/Microsoft/ms-tpm-20-ref/tree/master/Samples/ARM32-FirmwareTPM/ta_prod_signing_scripts